source/_posts/2018-10-26-Going-to-ServerSide-swift-2018.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Going to ServerSide.swift 2018
+date: 2018-10-26 18:00:00
+tags: vapor,swift,linux,swift4,vapor3,serversideswift,backend
+authorIds:
+- stso
+- re
+- sikk
+categories:
+- Vapor
+---
+
+## The first conference in the world to focus on server-side Swift
+We (the Vapor team at Nodes) were excited when we heard that a conference focusing on server-side Swift got announced. We did get the news a bit early since the conference is run by [Tim Condon](https://twitter.com/0xtim), [Martin Lasek](https://twitter.com/martinlasek) and [Steffen D. Sommer](https://twitter.com/steffendsommer) where Martin and Steffen both works at Nodes. Before this conference, we would look at conferences focused around Swift or more traditional "backend" conferences which most likely would focus on another language than Swift. Being able to go to a conference that mixes backend together with our programming language of choice is ideal to us.
+
+The whole Vapor team at Nodes met up in Berlin to participate in the conference and with this blog post we wanted to share a couple of highlights from the conference as well as giving some feedback on the conference in case any readers are considering to go next year.
+
+## Quick facts
+Some quick facts about ServerSide.swift 2018:
+- September 12-14 in Berlin (Adlershof)
+- One day of workshops. Four half-day workshops to choose from where two ran in parallel before lunch and the last two ran in parallel after lunch. This meant that you would be able to attend two in total.
+- Two days of talks. 8 talks per day and 18 speakers in total.
+- September 13 had a panel discussion after the last talk with five people from the community in the panel.
+- Website: [http://serversideswift.info](http://serversideswift.info) - videos should be available soon at the time of writing this post.
+
+## Talk highlights
+With 16 talks in total there has been a lot of knowledge to digest. Here are some highlights from the different talks we heard:
+
+### Vapor State Of The Union (Tanner Nelson)
+Since we at Nodes spend a lot of time using Vapor we were of course interested in the state of the union talk by Vapor's founder, Tanner Nelson. The talk went through the biggest changes from Vapor 2 to Vapor 3, being 1) how we deal with configs (read: no more JSON files), 2) how codable removes a lot of boiler plate code for us and 3) how Vapor is now built on top of SwiftNIO, making it possible for the Vapor team to delete a lot of code. The talk also gave a quick overview of the different packages that exist within the Vapor ecosystem and it provided a sneakpeak into what can be expected from Vapor Cloud 2 which [launched in alpha recently](https://medium.com/vapor-cloud/vapor-cloud-2-public-alpha-2d9ee14d0688). Last, the talk also touched upon the [Swift Server Work Group](https://swift.org/server/) and the [Swift Forums](https://forums.swift.org/c/development/server).
```

Comment:
```suggestion
Since we at Nodes spend a lot of time using Vapor we were of course interested in the state of the union talk by Vapor's founder, Tanner Nelson. The talk went through the biggest changes from Vapor 2 to Vapor 3, being 1) how we deal with configs (read: no more JSON files), 2) how Codable removes a lot of boiler plate code for us and 3) how Vapor is now built on top of SwiftNIO, making it possible for the Vapor team to delete a lot of code. The talk also gave a quick overview of the different packages that exist within the Vapor ecosystem and it provided a sneak peek into what can be expected from Vapor Cloud 2 which [launched in alpha recently](https://medium.com/vapor-cloud/vapor-cloud-2-public-alpha-2d9ee14d0688). Last, the talk also touched upon the [Swift Server Work Group](https://swift.org/server/) and the [Swift Forums](https://forums.swift.org/c/development/server).
```

Reply:
Suggested change highlighting broke here, codable -> Codable and sneakpeak to sneak peek